### PR TITLE
navbar

### PR DESCRIPTION
**Pull request by jdselby24**
navbar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+	<title>Josh Selby</title>
+	<link rel="stylesheet" href="https://use.typekit.net/gxz6jar.css">
```
Comment (sudokufan) [inline index.html:6]: what is this?

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
+/*        O__O
+          (oo)
+   /-------\/
+  / |     ||
+ *  ||----||
+    ~~    ~~
+Cow at Disneyland
+*/
```
Comment (sudokufan) [inline styles.css:8]: why? xD

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+/*               \_|_/
+                  (oo)
+           /-------\/
+          / |     ||
+         *  ||----||
+            ~~    ~~
+Cow visiting the Statue of Liberty
+*/
+
```
Comment (sudokufan) [inline styles.css:79]: are these for my benefit or yours? :)

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+	float: right;
+	display: none;
+}
+
+.hamburgerRow {
+  width: 30px;
+  height: 4px;
+  background-color: black;
+  margin: 6px 0;
+}
```
Comment (sudokufan) [inline styles.css:49]: these four lines aren't indented the way the others are, if that bothers you

**Review by sudokufan — COMMENTED** [review]
looks sick mate, love how you've labelled and organised everything

File: index.html (hunk 1 above)
Comment (jdselby24) [reply index.html:6]: link to external font, will add comment

File: styles.css (hunk 2 above)
Comment (jdselby24) [reply styles.css:8]: cow is jus chillin at disneyland

File: index.html (hunk 1 above)
Comment (jdselby24) [reply index.html:6]: external font

File: styles.css (hunk 3 above)
Comment (jdselby24) [reply styles.css:79]: nah they just out there

File: styles.css (hunk 4 above)
Comment (jdselby24) [reply styles.css:49]: thanks for pointing that out

**Comment by jdselby24** [comment]
@sudokufan cows are a great organisational tool :)

**Pull request merged**